### PR TITLE
Add default ClimateTypes per weather

### PR DESCRIPTION
**Pull request by Birdulon**
## Description
Loads in `WeatherExcelConfigData.json` to get default `ClimateType`s for weather IDs.
Updates `WeatherCommand` to take advantage of that, and lays some plumbing for future implementation of proper weather management.
image

## Type of changes
- [ ] Bug fix
- [x] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/main/resources/languages/ru-RU.json [hunk 1]
```diff
@@ -387,10 +388,10 @@
       "description": "Открывает все уровни башни"
     },
     "weather": {
-      "usage": "Применение: weather <тип_климата(Id погоды)> <тип_погоды(Id климата)>\nТипы погоды 0: Отсутствует, 1: Солнечная, 2: Пасмурная, 3: Дождливая, 4: Грозовая, 5: Снежная, 6: Туманная",
-      "success": "Тип климата был изменен на %s, тип погоды: %s.",
-      "invalid_id": "Некорректный ID.",
-      "description": "Изменяет погоду"
+      "description": "Изменяет погоду. Climate types sunny, cloudy, rain, thunderstorm, snow, mist.",
```
Comment (Birdulon) [inline src/main/resources/languages/ru-RU.json]: To anyone that translates this - `sunny`, `cloudy`, `rain`, `thunderstorm`, `snow`, and `mist` are literals that lookup the enum. Add translated terms next to them.

File: src/main/resources/languages/en-US.json [hunk 2]
```diff
@@ -387,10 +385,10 @@
       "description": "Unlock all levels of tower"
     },
     "weather": {
-      "usage": "Usage: weather <climate type(weatherId)> <weather type(climateId)>\nWeather types 0: None, 1: Sunny, 2: Cloudy, 3: Rain, 4: Thunderstorm, 5: Snow, 6: Mist",
-      "success": "Changed climate type to %s with weather type %s.",
-      "invalid_id": "Invalid ID.",
-      "description": "Changes the weather"
+      "description": "Changes the weather. Climate types sunny, cloudy, rain, thunderstorm, snow, mist.",
```
Comment (Tesutarin) [inline src/main/resources/languages/en-US.json]:
```suggestion
      "description": "Changes weather ID and climate type. Weather IDs can be found in WeatherExcelConfigData.json. Climate types: sunny, cloudy, rain, thunderstorm, snow, mist",
```

File: src/main/resources/languages/fr-FR.json [hunk 3]
```diff
@@ -387,10 +385,10 @@
         "description": "Débloque tous les couloirs de l'abysse"
       },
       "weather": {
-        "usage": "Usage: weather <climate type(weatherId)> <weather type(climateId)>\nTypes de météo 0: Aucun, 1: Ensoleillé, 2: Nuageux, 3: Pluvieux, 4: Orageux, 5: Neige, 6: Brouillard",
-        "success": "Le type de climat a été changé à %s avec le type de météo %s.",
-        "invalid_id": "ID invalide.",
-        "description": "Change la météo"
+        "description": "Change la météo. Climate types sunny, cloudy, rain, thunderstorm, snow, mist.",
+        "usage": "Utilisation: weather [weatherId] [climateType]\nClimate types sunny, cloudy, rain, thunderstorm, snow, mist.",
```
Comment (KingRainbow44) [inline src/main/resources/languages/fr-FR.json]: please add a colon between ‘types’ and ‘sunny’

**Review by KingRainbow44 — CHANGES_REQUESTED** [review]

File: src/main/resources/languages/fr-FR.json (hunk 3 above)
Comment (KingRainbow44) [reply src/main/resources/languages/fr-FR.json]: this goes for all languages btw, i only selected one

Comment (Birdulon) [reply src/main/resources/languages/fr-FR.json]: done in bcab81a

**Review by ghost — APPROVED** [review]

**Review by KingRainbow44 — APPROVED** [review]

**Comment by KingRainbow44** [comment]
please fix the merge conflict (i don't know how to handle it in this case lmao) and I'll merge the PR

**Pull request merged**